addon-builder/ember-cli-build.js

```diff
@@ -23,6 +23,11 @@ var importedCssFiles = [];
 // Files included via app.import need to end up in addon.js
 StubApp.prototype.import = function(assetPath, options) {
   options = options || {};
+  
+  if (typeof asset === 'object') {
```

Comment:
Shouldn't this be `if (typeof assetPath === 'object') {`??? `asset` vs `assetPath`